packages/components/pie-radio-group/src/index.ts

```diff
@@ -130,20 +130,39 @@ export class PieRadioGroup extends FormControlMixin(RtlMixin(LitElement)) implem
      * @param {Event} e - The slotchange event.
      * @private
      */
-    private _handleSlotChange (e: { target: HTMLSlotElement }): void {
+    private _handleLabelSlotChange (e: { target: HTMLSlotElement }): void {
         const childNodes = e.target.assignedNodes({ flatten: true });
         this._hasLabel = childNodes.length > 0;
     }
 
+    /**
+     * Ensures all newly added radio buttons are not tabbable and inherit the name property
```

Comment:
I'm shocked that @xander-marjoram hasn't pulled you up on making up the word `tabbable` 😆

Reply:
It stays ;)